<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_AddFrame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_NewProject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SaveProject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OpenProject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CloseProject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DeleteFrame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Left">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Right">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv_Canvas">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>80</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Animate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Export">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpriteTool">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSpriteTool"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
